app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controllers");
+const {
+  handlePsqlErrors,
+  handleCustomErrors,
+  handleServerErrors,
+} = require("./errors/error-handling");
+
+const app = express();
+
+app.get("/api/topics", getTopics);
+
+app.use(handlePsqlErrors);
```

Comment:
We're not yet using these error handlers (we're currently not catching or testing for any PSQL errors or custom errors). It's best practise to only include code in a PR that meets the need of that specific ticket (i.e. Task 2)
As these aren't applicable yet, they should be removed for this PR. 🔍